packages/cli/src/commands/deploy/render.js

```diff
@@ -36,29 +34,35 @@ export const builder = (yargs) => {
     )
 }
 
-export const handler = async ({ side, prisma, dm: dataMigrate, serve }) => {
+export const handler = async ({ side, prisma, dm: dataMigrate }) => {
   const paths = getPaths()
   let commandSet = []
-  if (side == 'api') {
+  if (side === 'api') {
     if (prisma) {
       commandSet.push('yarn rw prisma migrate deploy')
     }
     if (dataMigrate) {
       commandSet.push('yarn rw dataMigrate up')
     }
-    if (serve) {
-      commandSet.push('yarn rw serve api')
-    }
-  } else if (side == 'web') {
+  } else if (side === 'web') {
     commandSet.push('yarn')
     commandSet.push('yarn rw build web')
   }
 
-  execa(commandSet.join(' && '), {
-    shell: true,
-    stdio: 'inherit',
-    cwd: paths.base,
-    extendEnv: true,
-    cleanup: true,
-  })
+  if (commandSet.length) {
+    await execa(commandSet.join(' && '), {
+      shell: true,
+      stdio: 'inherit',
+      cwd: paths.base,
+      extendEnv: true,
+      cleanup: true,
+    })
+  }
+
+  if (side === 'api') {
+    await apiServerHandler({
+      port: getConfig().api?.port || 8911,
+      apiRootPath: '/',
+    })
+  }
```

Comment:
@SEANDOUGHTY @crcastle @dac09 
Questions about the RW API server config here:
1. **Port:** does it matter what the port is? And/or whether or not it respects redwood.toml web.port value?
2. **apiRootPath:** I'm assuming this does indeed need to be hardcoded here, correct?
3. **NODE_ENV:** Is the Render service automatically NODE_ENV set? If not, we should probably do a check and set if not exists similar to what we do over [here in this `rw serve` command code snippet](https://github.com/redwoodjs/redwood/blob/70ba72f00828b4ddcd1212b5f40c59af560ea2d1/packages/cli/src/commands/serve.js#L84-L87).

Reply:
## Part 2
> 2. Um, is this Render specific? I think this apiRootPath was a Redwood thing.

^^ It pertains to the render.yml for Redwood projects, specifically this part:
```
#####
# NOTE: replace destination api url after first deploy to Render
# example:
#   destination: https://myredwoodproject-api.onrender.com/*
#####
    destination: https://tdp-redwood-tutorial-test-api-r3h6.onrender.com/*
  - type: rewrite
    source: /*
    destination: /index.html
```

(@dac please add your thoughts here as well and confirm I understand the fastify config + Render settings correctly.) Typically during setup for a deploy target, the redwood.toml web.apiUrl value is set to be the endpoint required by the deploy host. In the case of Redwood, it's actually set to be `"/.redwood/functions"` [via the code here](https://github.com/redwoodjs/redwood/blob/007f82c2a649a579bf6e73c199e1405158e234a0/packages/cli/src/commands/setup/deploy/providers/render.js#L111). Effectively, though, the current settings do not respect project settings for apiUrl redwood.toml I can't think of a case where a project might need to set a specific apiUrl path, but right now they can't because it's hard coded. 

To Discuss:
- I think we should set the apiUrl to `"/"` [during `setup deploy render` here](https://github.com/redwoodjs/redwood/blob/007f82c2a649a579bf6e73c199e1405158e234a0/packages/cli/src/commands/setup/deploy/providers/render.js#L111)
- On it's own, the above will be backwards compatible because existing projects are overwriting. It's just providing the expected behavior.
- What I think we _should_ do is a breaking change. I.e. --> change the code above to be `apiRootPath: getConfig().web?.apiUrl || '/',` So we need to weigh the pros/cons of making this change.

## Part 3
> 3. If the NODE_VERSION is not set then ...

This is about the NODE_ENV being set to either  "production" or "development" or not set at all. Ideally, it should be set to "production" for the server.

Reply:
> What I think we should do is a breaking change. I.e. --> change the code above to be apiRootPath: getConfig().web?.apiUrl || '/', So we need to weigh the pros/cons of making this change.

I don't think we need this change actually!

- The redwood.toml `apiUrl` - is the url or path that the _web_ side is pointing to. `/.redwood/functions` is the correct value, because we then ask the user to go to the render dashboard and create an redirect 
![image](https://user-images.githubusercontent.com/1521877/151842177-cbc915f8-4b29-4bb1-80cb-5e9cc6440dc1.png)
apiUrl in this case only configures the web side, and we use a proxy to avoid CORS issues

- The cli `apiRootPath`, is something different. I think there was a reason this was added, but having it locked to `/` for a render deploy is the desired behaviour I believe. This should not be the same value as `getConfig().web?.apiUrl`

Reply:
>  having it locked to / for a render deploy is the desired behaviour

^^ This makes sense to me. I just wanted to confirm:
1. we can't foresee a reason if/how a user would want to override with custom config, correct?
2. either way, we should update the `setup deploy render` to set apiUrl to `"/"` to be consistent (logically) with the API path during production deploy. (Again, I understand it's not applicable mechanically. But in all other cases we update to match what the deploy host provider is going to use for the API  path — so I'm just suggesting consistent behavior.)

Reply:
Confirming NODE_ENV=production per this check I did from the shell tab of the Render Dashboard. Looks good but please confirm otherwise:
<img width="604" alt="Screen Shot 2022-01-31 at 3 57 51 PM" src="https://user-images.githubusercontent.com/2951/151893152-a7182d0a-b19b-4d52-b6e8-c7013d5c43fa.png">